assets/javascripts/needleeditor.js

```diff
@@ -19,6 +19,8 @@ NeedleEditor.prototype.init = function () {
     return;
```

Comment:
It would be great if the canvas width/height would adjust itself when switching the screenshot via the "Take image from:" combo box. I suppose all you need to do for that is moving your new lines before that if block so it is also executed in case of the early return.

Reply:
OK, that should be done.